src/dbos-runtime/config.ts

```diff
@@ -70,9 +75,15 @@ export function loadConfigFile(configFilePath: string): ConfigFile {
   }
 }
 
-export function writeConfigFile(configFile: ConfigFile, configFilePath: string) {
+/**
+ * Writes a ConfigFile or YAML.Document object to configFilePath.
+ * @param {ConfigFile | Document} configFile - The config file to be written. 
+ * @param {string} configFilePath - The path to the config file to be written to.
+ * @param {boolean} document - Whether the config file is a YAML.Document object. Defaults to false.
+ */
+export function writeConfigFile(configFile: ConfigFile | YAML.Document, configFilePath: string, document: boolean=false) {
```

Comment:
This function is only used by `configure.ts` so I think we can simply change the interface to YAML.Document.